src/ray/ray_exported_symbols.lds

```diff
@@ -27,3 +28,8 @@
 *init_raylet*
 *Java*
 *JNI_*
+#boost
+*boost*;
+*ontop_fcontext*;
+*make_fcontext*;
+*jump_fcontext*;
```

Comment:
What those symbols is for:
```
*ontop_fcontext*;
*make_fcontext*;
*jump_fcontext*;
```

Reply:
removed, it's only used when compiled in debug mode.